### DISCUSSION TITLE
Modifying Frameworks

### DISCUSSION THREAD

**Discussion by sesem738** [isaac-sim/IsaacLab#1132] [Q&A]
I'm looking to implement an asymmetric actor-critic architecture using transformer models in the RL frameworks. Specifically, I'd like to know how to achieve this using SKRL or RL-Games. I've successfully implemented this approach in my custom training framework, but I'm unsure how to translate it to these libraries. Can you provide guidance on integrating asymmetric architectures and transformer models within SKRL or RL-Games?

**Comment by KyleM73** [comment]
I can't speak to SKRL but RL-Games has an active discord server where the maintainers answer questions, so I would recommend asking there.

https://discord.gg/hnYRq7DsQh
